tests/karate/src/test/java/be/rollCall/createRollCall.feature

```diff
@@ -46,6 +45,7 @@ Feature: Create a Roll Call
   # an empty roll call name should result in an error message from the backend.
   Scenario: Roll Call Creation with empty name should return an error code
     Given call read('classpath:be/utils/simpleScenarios.feature@name=valid_lao')
+    * def rollCallEmptyName = ""
```

Comment:
Is this variable used ?
I cannot find it.

Reply:
Forgot to erase is, thank you for noticing